draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -246,76 +312,106 @@ Internet-Draft      Network Slice Topology Data Model         March 2023
 
 2.2.  ACTN for Network Slicing
 
-   Since ACTN topology data models are based on the network topology
-   model defined in [RFC8345], the augmentations defined in this
-   document are effective augmentations to the ACTN topology data
-   models, resulting in making the ACTN framework [RFC8453] and data
-   models [I-D.ietf-teas-actn-yang] capable of slicing networks with the
-   required network characteristics.
+   Since ACTN topology data models defined in [RFC8795] are based on the
```

Comment:
I would avoid to make 1 to 1 association between ACTN topology and RFC8795. I would keep the old wording here talking about general ACTN topology data models : " Since ACTN topology data models are based on the network topology model defined in [RFC8345], the augmentations defined in this document , due to multi-inheritance, are effective augmentation to the ACTN topology data models"

Reply:
Text proposal:
~~~
   Since ACTN topology data models are based on the network topology model
   defined in {{!RFC8345}}, the augmentations defined in this document, 
   due to multi-inheritance, are effective augmentation to the ACTN topology
   data models, resulting in the ACTN framework {{?RFC8453}} and data
   models {{?I-D.ietf-teas-actn-yang}} capable of supporting network slicing
   over traffic-engineering (TE) enabled networks.
~~~